ci-operator/step-registry/openstack-k8s-operators/test/build-deploy/openstack-k8s-operators-test-build-deploy-workflow.yaml

```diff
@@ -5,6 +5,7 @@ workflow:
     - ref: openstack-k8s-operators-gather
     test:
     - ref: openstack-k8s-operators-build
+    - ref: openstack-k8s-operators-kuttl
```

Comment:
Since you are changing files in step registry, this will affect all other build-deploy jobs.
So, the best way would be to create a new workflow, called build-deploy-kuttl, and add the kuttl step in this workflow.
With that, you can go on each operator condig, which already have kuttl tests and update the workflow.
It will also make easier to identify, by the name of the job, which repo already runs kuttl test.